migrations/20190602121612-create-table-point-of-interest.js

```diff
@@ -0,0 +1,58 @@
+/* eslint-disable func-names */
+
+
+let dbm;
+// eslint-disable-next-line no-unused-vars
+let type;
+// eslint-disable-next-line no-unused-vars
+let seed;
+
+/**
+  * We receive the dbmigrate dependency from dbmigrate initially.
+  * This enables us to not have to rely on NODE_PATH.
+  */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+exports.up = function (db) {
+  return db.createTable('pointofinterest', {
```

Comment:
camelCase or other naming convention for 'pointofinterest'

Reply:
Done